pkg/algorithm/cell.go

```diff
@@ -24,6 +24,7 @@ package algorithm
 
 import (
 	"fmt"
+
 	"github.com/microsoft/hivedscheduler/pkg/api"
 	"k8s.io/klog"
 )
```

Comment:
go format the project

Reply:
Use Golang 1.12.6 for your local build and dev

---
In reply to: [451362395](https://github.com/microsoft/hivedscheduler/pull/27#discussion_r451362395) [](ancestors = 451362395)

Reply:
it's the result of lint and format

Reply:
reference: https://github.com/golang/go/wiki/CodeReviewComments#imports

Reply:
Pls use  Golang 1.12.6 to reformat and do not use lint. This is current standard

Reply:
updated